### PR TITLE
(PA-2536) - (PA-2600), (PA-2601), (PA-2608) Docs/split doc md 

### PR DESCRIPTION
**Pull request by bertizman**

### CODE REVIEW

File: documentation/system_specific.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    - [Data extraction & connection to source system](#Data-extraction-&-connection-to-source-system)
+    - [Input data](#Input-data)
+- [Connector configuration](#Connector-configuration)
+- [Connector transformation design](#Connector_transformation_design)
+    - [Entities](#Entities)
+    - [Activities](#Activities)
+    - [Attributes](#Attributes)
+    - [Design flowchart](#Design-flowchart)
+    - [Design details](#Design-details)
+- [How to extend the connector](#How-to-extend-the-connector)
```
Comment (cverhoef) [inline documentation/system_specific.md:20]: Minor thing: section is still called 'Adding output attributes'.

File: documentation/process_specific.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# \<Connector title>
+
+[REMOVE THIS: The intended audience of this documentation is a developer persona that intends to use/deploy this connector. Please keep this in mind when writing this documentation.]
+
+## Table of contents
+- [Process description](#Process-description)
```
Comment (cverhoef) [inline documentation/process_specific.md:6]: Other minor thing: can we name the section 'Process' and have the first subsection called 'Description'. This is the only part in the documentation template that we do not have a subsection.

**Review by cverhoef — APPROVED** [review]
Approved with 2 small comments.

File: documentation/process_specific.md (hunk 2 above)
Comment (bertizman) [reply documentation/process_specific.md:6]: Subsection is added.

**Pull request merged**